### ISSUE TITLE
 System.UnauthorizedAccessException

### ISSUE THREAD

**Issue by saran2250** [gsoft-inc/ansible-role-azure-devops-agent#43]
Hi am getting this error while configuring agent, any help..
"Unhandled exception. System.UnauthorizedAccessException: Access to the path '/home/az_devops_agent/agent/_diag/Agent_20210408-131733-utc.log' is denied.\n ---> System.IO.IOException: Permission denied

**Comment by yohanb** [comment]
hi @saran2250 .  What kind config do you have for the ansible role?
Did you check what permissions are set on the log file?
What happens when you execute the playbook again?

**Comment by krknopp** [comment]
I had this same problem.  But I got it after running the role and getting the error requiring the `acl` package to be installed on the target system.  After installing that package, I got the above error, as it moved down the role farther.

To fix it, I removed the folders created by this role, in the case of the OP, `/home/az_devops_agent` and then reran the role.  It worked after that. 

I think it's be a good idea for the role to ensure that `acl` is installed as I don't think it is by default on Ubuntu 20.04
